### DISCUSSION TITLE
Tiddlywiki "server"-command wording in help

### DISCUSSION THREAD

**Discussion by toraxmalu** [TiddlyWiki/TiddlyWiki5#7718] [Ideas]
Tiddlywiki on command line is marking in the help the wording "* server: Provides an HTTP server interface to TiddlyWiki (deprecated in favour of the new listen command)".

Maybe to mark it better, replacing the help text with somethinh like "* server: deprecated, see listen-command; Provides an HTTP server interface to TiddlyWiki (only for backwards compatibaly)"

**Comment by pmario** [comment]
@Jermolene -- Can you move this one into the issues. So we do not forget about it?

**Reply by linonetwo** [reply]
I think a PR is better...

**Comment by Jermolene** [comment]
Thanks @toraxmalu – I created an issue #7724 and pushed a fix.

**Comment by toraxmalu** [comment]
You're welcome. Played a little with tiddlywiki again and noticed it…
